RaidBot/Entities/Mons.cs

```diff
@@ -67,5 +68,63 @@ public static class Mons {
             )}
```

Comment:
Strange - I thought this was already merged into master. Maybe it's just GH playing with us.